src/HvLightPipe.c

```diff
@@ -32,6 +32,9 @@
 #elif HV_WIN
 // https://msdn.microsoft.com/en-us/library/windows/desktop/ms684208(v=vs.85).aspx
 #define hv_sfence() _WriteBarrier()
+#elif ARM_CORTEX
+    #include <arm_acle.h>
+    #define hv_sfence() __dmb(0xE) /* _ARM_BARRIER_ST */
```

Comment:
This is already defined on Line 27 [0], are `__arm__ ` and `__ARM_ACLE` not defined on the OWL platform?

[0]: (https://github.com/enzienaudio/heavy/pull/103/commits/9da945c73dabef5edb40c3ac1608b9660d3193d3#diff-9b75b5f8f69ea85f9a93de92913e99faL27)